contracts/contracts/RaidenMicroTransferChannels.sol

```diff
@@ -88,10 +95,22 @@ contract RaidenMicroTransferChannels {
         challenge_period = _challenge_period;
     }
 
+    /// @dev Sets the address for the latest contract version
```

Comment:
Hmm I am a bit confused. Why add a version setter? Once the code of the contract changes the version should also change. At least the `PATCH` part from `MAJOR.MINOR.PATCH`.

Reply:
In case someone finds this contract and wants to use it and is not sure if it is the last version, it's nice to be able to check this and have a way to get the last contract version deployed.
Of course, this means taking care of updating all deployed versions when deploying a new one - so the setter will be used in the deployment process.

This was the reasoning. More details in the comments here https://github.com/raiden-network/microraiden/issues/99

Reply:
@LefterisJP , I will assume a misunderstanding (version vs. address) and will merge this PR for now. If not, please open a issue.
Removed the redundant getter though.

Reply:
Hmm I don;t really think this is required but if you would like to have it then you can have it. I rather think just having an ENS record pointing to the latest contract should be sufficient